### ISSUE TITLE
Feature Request: Add an option to delay screenshot being sent to Discord

### ISSUE THREAD

**Issue by UmaLPZ** [pajlads/DinkPlugin#339]
Love the plugin and thank you for adding the npc kill count to the loot notifications.

Is it possible to add an option to delay the notification from being sent to Discord or does this option already exist? I'm in a clan event right now and we use this plugin for backup screenshots for loot drops. Unfortunately, that involves having the loot notification that appears in chat also appear for the drop to count. The plugin will send the screenshot before the chat shows the drop, so we still have to manually take a screenshot. The plugin is helpful is still very helpful for keeping track of drops for the event regardless. An option to add a small 1 to 2 second delay would be appreciated.

Thank you.

**Comment by iProdigy** [comment]
Could you specify *which* NPCs lead to loot screenshots that are too early, in your experience?

**Comment by UmaLPZ** [comment]
> Could you specify _which_ NPCs lead to loot screenshots that are too early, in your experience?

Ooops been out of town recently. It happened to me when killing vorkath, calvarion, demonic gorillas, and spiritual mages. Others in my clan had it happen at places like duke and the other dt2 bosses.

**Comment by iProdigy** [comment]
Do you have `Valuable drop` chat notifications enabled? We're not able to reproduce this issue for vorkath/duke

https://i.ivr.fi/runelite/fae185d191e9.png
https://i.ivr.fi/runelite/538a5f12adbe.png

**Comment by Felanbird** [comment]
My assumption was this problem was related to some other hub plugin or just general lag, I personally never have it for any boss except Whisperer, vorkath and duke never; pajlada did have it happen for a ring drop at dag kings but it was the opposite, he had already picked up the ring before the notif went off, not before.

**Comment by iProdigy** [comment]
other dt2 bosses look good

https://i.ivr.fi/runelite/c30528f42146.png
https://i.ivr.fi/runelite/758c8dd4a076.png
https://i.ivr.fi/runelite/eb7a46acc79b.png

@UmaLPZ Are you expecting the clan loot notification to be in the screenshot instead of the valuable drop chat message? (the clan message has greater delay)

**Comment by UmaLPZ** [comment]
Yes, I have valuable drop chat notifications enabled and yes, the requirement was to have the clan loot notification also be in the screenshot. There was an issue with the Wise Old Man plugin that was causing some clients to crash so that might have had something to do with it. I don't know what other plugins may have caused an issue since not everyone in my clan has the same plugins installed. Some of the screenshots from my clan members do show higher ping than me but still get the screenshot to include the clan notification.

**Comment by iProdigy** [comment]
Do your screenshots include the valuable drop chat message?

I'm apprehensive to delay loot notifs until the clan message arrives since it could be possible to grab the items before the clan message arrives, yielding a screenshot without the ground item present

**Comment by UmaLPZ** [comment]
Yes, they all include the valuable drop message. For the clan event we were allowed to post screenshots of the item in our inventory as long as we included the clan message and it was soon after the drop. We primarily needed the codeword displayed by Wise Old Man plugin and the clan message.

Would it be possible to have the delay be optional and adjustable? Since this doesn't affect the valuable drop message being left out and is only really needed for some users or clan events. The delay isn't needed overall, but it comes it makes clan events much easier to track.
